pkg/ekcoops/operator.go

```diff
@@ -221,3 +228,31 @@ func shouldUseNodeForStorage(node corev1.Node, rookStorageNodesLabel string) boo
 	}
 	return false
 }
+
+//TODO approve the csrs if the SIGNERNAME is kubernetes.io/kubelet-serving
+
+func (o *Operator) reconcileCertificateSigningRequests() error {
+	csrList, err := o.client.CertificatesV1().CertificateSigningRequests().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		if !kuberneteserrors.IsNotFound(err) {
+			return errors.Wrap(err, "csrs do not exist")
+		}
```

Comment:
I think your logic here is backwards ("CSRs do not exist" seems like an error message for if the error IS 'is not found'

and also IMO if it is a not found error we should return nil, since there's nothing to approve